src/site/content/en/blog/lighthouse-whats-new-6.0/index.md

```diff
@@ -0,0 +1,490 @@
+
+- [New metrics](#new-metrics)
+- [Performance score update](#score)
+- [New audits](#new-audits)
+- [Lighthouse CI](#ci)
+- [Renamed Chrome DevTools panel](#devtools)
+- [Mobile emulation](#emulation)
+- [Browser extension](#extension)
+- [Budgets](#budgets)
+- [Source location links](#source-location)
+- [On the horizon](#horizon)
+- [Thank you!](#thanks)
+
+## New metrics {: #new-metrics }
+
+<figure class="w-figure">
+  <img class="w-screenshot" src="lighthouse-new-metrics.png" alt="Lighthouse 6.0 metrics." width="600">
+</figure>
+
+Lighthouse 6.0 introduces three new metrics to the report. Two of these new metrics–Largest
+Contentful Paint (LCP) and Cumulative Layout Shift (CLS)–are lab implementations of [Core Web
+Vitals](https://web.dev/vitals/).
+
+### Largest Contentful Paint (LCP) {: #lcp }
+
+[Largest Contentful Paint (LCP)](https://www.web.dev/lcp/) is a measurement of perceived loading
+experience. It marks the point during page load when the primary–or "largest"–content has loaded and
+is visible to the user. LCP is an important complement to First Contentful Paint (FCP) which only
+captures the very beginning of the loading experience. LCP provides a signal to developers about how
+quickly a user is actually able to see the content of a page. An LCP score below 2.5 seconds is
+considered 'Good.'
+
+For more information, [watch this deep-dive on LCP](https://youtu.be/diAc65p15ag) by Paul Irish.
+
+### Cumulative Layout Shift (CLS) {: #cls }
+
+[Cumulative Layout Shift (CLS)](https://www.web.dev/cls/) is a measurement of visual stability. It
+quantifies how much a page's content visually shifts around. A low CLS score is a signal to
+developers that their users aren't experiencing undue content shifts; a CLS score below 0.10 is
+considered 'Good.'
+
+CLS in a lab environment is measured through the end of page load. Whereas in the field, you can
+measure CLS up to the first user interaction or including all user input.
+
+For more information, [watch this deep-dive on CLS](https://youtu.be/zIJuY-JCjqw) by Annie Sullivan.
+
+### Total Blocking Time (TBT) {: #tbt }
+
+[Total Blocking Time (TBT)](https://www.web.dev/tbt/) quantifies load responsiveness, measuring the
+total amount of time when the main thread was blocked long enough to prevent input responsiveness.
+TBT measures the total amount of time between First Contentful Paint (FCP) and Time to Interactive
+(TTI). It is a companion metric to TTI and it brings more nuance to quantifying main thread activity
+that blocks a user's ability to interact with your page.
+
+Additionally, TBT correlates well with the field metric [First Input Delay](https://web.dev/fid/)
+(FID), which is a Core Web Vital.
+
+## Performance score update  {: #score }
+
+The [performance score in Lighthouse](https://web.dev/performance-scoring/) is calculated from a
+weighted blend of multiple metrics to summarize a page's speed. The 6.0 performance score formula
+follows.
+
+<style>
```

Comment:
I was wondering why y'all needed this? Maybe because the table is missing the outer `w-table-wrapper` class?